app/src/__utils__/safe-integer.ts

```diff
@@ -0,0 +1,78 @@
+type numeric = number | bigint;
+
+export class SafeInteger {
+	private _value: numeric;
```

Comment:
I dont think we need an underscore in the variable name here

```suggestion
	private value: numeric;
```

Reply:
I did set it private so we can't modify it outside of the class. But can't have a getter with the same name as a variable name.